core/src/main/java/hudson/FilePath.java

```diff
@@ -1105,23 +1104,28 @@ protected void after() {}
      * @since 1.522
      */
     public <V> Callable<V,IOException> asCallableWith(final FileCallable<V> task) {
-        return new Callable<V,IOException>() {
-            @Override
-            public V call() throws IOException {
-                try {
-                    return act(task);
```

Comment:
[Whitespace-reduced diff](https://github.com/jenkinsci/jenkins/pull/3333/files?w=1) is easier to review.